src/plugins/grpc/client.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const FORMAT_HTTP_HEADERS = require('opentracing').FORMAT_HTTP_HEADERS
+const Tags = require('opentracing').Tags
+
+function getMethodType (definition) {
+  if (definition.requestStream) {
+    if (definition.responseStream) {
+      return 'bidi'
+    }
+
+    return 'client_stream'
+  }
+
+  if (definition.responseStream) {
+    return 'server_stream'
+  }
+
+  return 'unary'
+}
+
+function patch (grpc, tracer, config) {
+  if (config.client === false) return
+
+  config = config.client || config
+
+  function datadogInterceptor (options, nextCall) {
+    const path = options.method_definition.path
+    const methodType = getMethodType(options.method_definition)
+    const scope = tracer.scopeManager().active()
+    const span = tracer.startSpan('grpc.client', {
```

Comment:
This should be `grpc.request` to be consistent with other integrations we well as gRPC's own terminology.